eng/revapi.json

```diff
@@ -0,0 +1,25 @@
+[
+    {
+        "extension": "revapi.java",
+        "configuration": {
+            "filter": {
+                "packages": {
+                    "regex": true,
+                    "exclude" : ["com\\.azure\\..+\\.implementation(\\..+)?"]
+                }
+            }
+        }
+    },
+    {
```

Comment:
We might use reclassify extension in future.   More information here https://revapi.org/modules/revapi-java/differences.html

Reply:
This file can be updated in the future. I didn't add this because we don't have any breaking changes.